tailblocks/src/App.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react'
-import Header from './components/Header'
-import Hero from './components/Hero'
-import Description from './components/Description'
-import Solution from './components/Solution'
-import Technology from './components/Technology'
-import Footer from './components/Footer'
-function App() 
+import React from 'react';
+import Home from './Pages/Home';
+import About from './Pages/About';
+import Team from './Pages/Team';
+import Contact from './Pages/Contact';
+
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+
+import Header from './components/Header';
+function App() {
   return (
     <>
-      <Header />
-      <Hero />
-      <Description />
-      <Solution />
-      <Technology />
-      <br />
-      <Footer />
+      <Router>
+        <Header />
+        <Switch>
+          <Route exact path='/'>
+            <Home />
+          </Route>
+          <Route path='/about'>
+            <About />
+          </Route>
+          <Route path='/team'>
+            <Team />
+          </Route>
+          <Route path='/contact'>
```

Comment:
I think we should add an error page too. Right now it just shows an empty component.

Reply:
Done. added 404 error page.